### ISSUE TITLE
ORDS image overwrites ords_params.properties

### ISSUE THREAD

**Issue by stif** [oracle/docker-images#1964]
There is already a active Issue about the outdated ORDS docker image: #1888 

> ORDS 18.4 is no longer available for download, the oldest version available is 19.4.6, so unless I'm missing something there is currently no way we can make use of this container.

Since it does not get any love since month and because i think i found a problem with recent ORDS versions i created a new Issue.

These are the steps i followed: 

I have added a file `Checksum.21.1.1` inside [dockerfiles](https://github.com/oracle/docker-images/tree/main/OracleRestDataServices/dockerfiles) with the content of
`a6afba1b823ee77a10c365f240a67812  ords-21.1.1.116.2032.zip`

The `./buildDockerImage.sh` command finishing sucessfully resulting in a `oracle/restdataservices` image

For startup of db and ords i created a docker-compose file

```
database:
    image: oracle/database:19.3.0-se2
    environment:
      - ORACLE_SID=ORCLCDB
      - ORACLE_PDB=ORCLPDB1
      - ORACLE_PWD=oracleDBpassword
      - ORACLE_EDITION=standard
      - ORACLE_CHARACTERSET=AL32UTF8
    volumes:
      - /root/oracle/oradata:/opt/oracle/oradata # persistent oracle database data.
    restart: always
    networks:
      oracle:
        ipv4_address: 192.168.20.20
ords:
    image: oracle/restdataservices:21.1.1
    environment:
      - ORACLE_HOST=192.168.20.20
      - ORACLE_PORT=1521
      - ORACLE_SERVICE=ORCLPDB1
      - ORACLE_PWD=oracleDBpassword
      - ORDS_PWD=testORDS
      - CONTEXT_ROOT=ords
    ports:
      - 8888:8888
    networks:
      oracle:
        ipv4_address: 192.168.20.31
    volumes:
      - ./ords:/opt/oracle/ords/config/ords
```

the container also starts up without any problem:
```
ords_1       | 2021-06-09T14:46:40.562Z INFO        HTTP and HTTP/2 cleartext listening on host: localhost port: 8888
ords_1       | 2021-06-09T14:46:40.654Z INFO        The document root is serving static resources located in: /opt/oracle/ords/doc_root
ords_1       | 2021-06-09T14:46:42.043Z INFO        No pools configured yet
ords_1       | 2021-06-09T14:46:43.154Z INFO        Oracle REST Data Services initialized
ords_1       | Oracle REST Data Services version : 21.1.1.r1162032
ords_1       | Oracle REST Data Services server info: jetty/9.4.40.v20210413
```
But it has no connection to the Database and i think i know why:

the content of the file [ords_params.properties.tmpl](https://github.com/oracle/docker-images/blob/main/OracleRestDataServices/dockerfiles/ords_params.properties.tmpl) is as following:

```
rest.services.ords.add=true
db.hostname=###ORACLE_HOST###
db.port=###ORACLE_PORT###
db.servicename=###ORACLE_SERVICE###
plsql.gateway.add=false
rest.services.apex.add=false
rest.services.ords.add=true
standalone.http.port=8888
standalone.mode=false
standalone.use.https=true
sys.user=SYS
sys.password=###ORACLE_PWD###
user.public.password=###ORDS_PWD###
```

whereas when entering the container with `docker exec -it oracle_ords_1 /bin/bash` the content of the file `/opt/oracle/ords/params/ords_params.properties` is:

```
db.hostname=
db.port=
db.servicename=
db.sid=
db.username=APEX_PUBLIC_USER
migrate.apex.rest=false
rest.services.apex.add=
rest.services.ords.add=true
schema.tablespace.default=SYSAUX
schema.tablespace.temp=TEMP
standalone.http.port=8080
standalone.static.images=
user.tablespace.default=SYSAUX
user.tablespace.temp=TEMP
```
Would be great to have a working ORDS Docker container anytime soon!

cheers,
stif

**Comment by gvenzl** [comment]
Hey @stif, a quick update on this as well as #1888.

We are in the process of building and providing ORDS Docker images up on [container-registry.oracle.com](https://container-registry.oracle.com).
The build files here will be updated and form the basis for the ORDS images themselves.
Once that is in place, I expect many if not all of the issues here to be addressed.
